### ISSUE TITLE
Cursor visibility on a vertical canvas

### ISSUE THREAD

**Issue by estebandelpino** [BlankSourceCode/obs-zoom-to-mouse#31]
Hi! Loving the script so far! – I'm currently working on configuring it to operate on a vertical canvas. However, I'm encountering some challenges due to using an ultrawide monitor (2560x1080). As a result, a significant portion of the image gets cropped out on the vertical canvas area.

I'm specifically struggling with making the cursor visible on the canvas even when it reaches the edges of the source. Is there a way to configure this?

I've attached a video to provide a clearer understanding of the issue. Any assistance you can provide would be greatly appreciated!

– Esteban

[Zoom To Mouse issue.webm](https://github.com/BlankSourceCode/obs-zoom-to-mouse/assets/160371036/f8fbe74b-1f6e-455d-86c2-3332a2e143db)

**Comment by BlankSourceCode** [comment]
@estebandelpino There are some known issues with ultrawide monitors that I've not managed to fix yet.

Could you take a look at the workaround posted in #30 and try modifying the script to match then let me know if it works for you?

**Comment by estebandelpino** [comment]
That did the trick! Thank you so much for pointing the workaround. 😁

**Comment by ibrah3m** [comment]
> That did the trick! Thank you so much for pointing the workaround. 😁

do you have issue when you switch the main obs scene the zoom stop work8 on vertical plugin?
I just got this looking for solutions
@BlankSourceCode

**Comment by BlankSourceCode** [comment]
Are you using a plugin to do vertical videos? Or are you just resizing the canvas? Can you link whatever the plugin is (if you are using one), so I can see what it does.

**Comment by ibrah3m** [comment]
> Are you using a plugin to do vertical videos? Or are you just resizing the canvas? Can you link whatever the plugin is (if you are using one), so I can see what it does.
> 
> 

yes I do use plugin and this plugin allows sharing theirs scenes with main obs also 
https://aitum.tv/products/vertical

**Comment by ibrah3m** [comment]
any updates ?
